scripts/qemu-dm-wrapper

```diff
@@ -61,43 +61,21 @@ def enable_core_dumps():
         setrlimit(RLIMIT_CORE, (limit, hardlimit))
         return limit
 
-def qemu_dm_upstream():
-    return '/usr/lib64/xen/bin/qemu-wrapper'
-
-def qemu_dm_trad():
+def main(argv):
+    qemu_env = os.environ
     qemu_dm_list = ['/usr/lib/xen/bin/qemu-dm',
                     '/usr/lib64/xen/bin/qemu-dm',
                     '/usr/lib/xen-4.1/bin/qemu-dm',
```

Comment:
At some point this list should be pruned (I actually don't know why we don't just have a single binary to call). But that can be done later.